spec/resources/dirty_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+RSpec.describe LedgerSync::Resource do
+  let(:resource) { LedgerSync::Customer.new }
+
+  it do
+    expect(resource).not_to be_changed
+    resource.external_id = :asdf
+    expect(resource).to be_changed
+    expect(resource.changes).to have_key('external_id')
```

Comment:
Maybe test that the change shows up exactly. Like test the key/value pair

Reply:
Added!  Ready to merge when approved.